isis/src/base/objs/Camera/Camera.cpp

```diff
@@ -2246,22 +2233,18 @@ namespace Isis {
    * @return @b double Off Nadir Angle
    */
   double Camera::OffNadirAngle() {
-    NaifStatus::CheckErrors();
-
     // Get the xyz coordinates for the spacecraft and point we are interested in
     double coord[3], spCoord[3];
     Coordinate(coord);
     instrumentPosition(spCoord);
 
     // Get the angle between the 2 points and convert to degrees
-    double a = vsep_c(coord, spCoord) * 180.0 / PI;
+    double a = SensorUtilities::sepAngle(coord, spCoord) * RAD2DEG;
```

Comment:
This is causing errors in deltack, I have updated the deltack truth data but I am unsure if this is the correct action see PR #5081